core/src/main/java/org/apache/iceberg/rest/responses/LoadTableResponse.java

```diff
@@ -74,12 +83,17 @@ public Map<String, String> config() {
     return config != null ? config : ImmutableMap.of();
   }
 
+  public List<Credential> credentials() {
+    return credentials != null ? credentials : ImmutableList.of();
+  }
+
   @Override
   public String toString() {
     return MoreObjects.toStringHelper(this)
         .add("metadataLocation", metadataLocation)
         .add("metadata", metadata)
         .add("config", config)
+        .add("credentials", credentials)
```

Comment:
I'm not sure we be adding this to the `toString`? Bit concerned if someone is inadvertently logging load table response then they'd be logging sensitive info